src/components/CustomClient.js

```diff
@@ -46,15 +49,15 @@ class CustomClient extends React.Component {
   };
 
   submit = (event) => {
-    const { toLoad } = this.state;
+    let { toLoad } = this.state;
```

Comment:
Echoing @dylanesque 's question of why these were changed to `let`s

Reply:
See below.